model/model.go

```diff
@@ -284,6 +284,15 @@ type PartitionInfo struct {
 	Definitions []PartitionDefinition `json:"definitions"`
 }
 
+func (pi *PartitionInfo) GetNameByID(id int64) string {
+	for _, def := range pi.Definitions {
+		if id == def.ID {
+			return def.Name.L
+		}
+	}
+	return ""
```

Comment:
Should we return error here?

Reply:
Not necessary.
The only place we need to know the partition name is `explain`, while `explain` doesn't care `GetNameByID` fail.

Reply:
Can it be changed to `notFind`?  empty strings that don't make sense are bad.

Reply:
empty string means we have to check the code and fix bug, `notFind` gives no more information than that.

```
table: t, partition: 
```

It's a bug!

```
table: t, partition: notFind
```

It's a bug!